test/modes/GalacticBridge.jl

```diff
@@ -0,0 +1,111 @@
+using Turing
+using Optim
```

Comment:
As mentioned above, I don't think Optim should be supported explicitly anymore. Instead users should just use GalacticOptim (whose default backend is Optim).
```suggestion
```

Reply:
Hm, I don't want to do this shift quite yet -- lots of people use the MLE/MAP functionality as it currently exists, so I don't want to break everyone's code without offering a nice way to use GalacticOptim instead. Maybe we could add some convenience functions for `optimize` that map to GalacticOptim?

Reply:
I agree, I have kept the `optimize` function in a separate file in the subsequent commit, which is conditionally loaded when `Optim.jl` is loaded.